permission-portal-frontend/client/src/screens/Login.js

```diff
@@ -85,6 +88,12 @@ const SignInFormBase = observer(
           </div>
           <ForgotPasswordModal hidden={!this.state.showPassModal} onClose={this.hideModal} />
         </div>
+        <Toast
+          open={this.state.showToast}
+          onClose={() => this.setState({ showToast: false })}
```

Comment:
Should we make this the default behavior of Toasts? This would resolve #181.

Reply:
I don't think that quite makes sense. The `Toast` would need to somehow get `this` passed in order to `setState` for its parent component.

Reply:
(Feel free to prove me wrong in another PR, though, perhaps I'm missing something)